modules/friendlywelcome.js

```diff
@@ -471,6 +472,11 @@ Twinkle.welcome.templates = {
 		linkedArticle: false,
 		syntax: "{{subst:welcome-anatomy}} ~~~~"
 	},
+	"welcome-athletics": {
+		description: "welcome for users with an apparent interest in athletics (track and field) topics",
+		linkedArticle: false,
+		syntax: "{{subst:welcome-athletics}}"
+	}
```

Comment:
Please add a comma after this curly bracket.

Reply:
Apologies, done!